docs/update_openshift.md

```diff
@@ -0,0 +1,217 @@
+                ```
+
+        2. [Apply a patch :octicons-link-external-16:](https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/) to update the `initContainer.image` option of your cluster Custom Resource with this value. Supposing that your cluster name is `cluster1`, the command should look as follows:
+
+            ``` {.bash data-prompt="$" }
+            $ kubectl patch pxc cluster1 --type=merge --patch '{
+                "spec": {
+                   "initContainer": { "image": "registry.connect.redhat.com/percona/percona-xtradb-cluster-operator@sha256:4edb5a53230e023bbe54c8e9e1154579668423fc3466415d5b04b8304a8e01d7" }
+                }}'
+            ```
+
+    2. Now you can actually update the Operator via the [Operator Lifecycle Manager (OLM) :octicons-link-external-16:](https://docs.redhat.com/en/documentation/openshift_container_platform/4.2/html/operators/understanding-the-operator-lifecycle-manager-olm#olm-overview_olm-understanding-olm) web interface.
+
+        Login to your OLM installation and list installed Operators for your Namespace to see if there are upgradable items:
+
+        ![image](assets/images/olm4.svg)
+
+        Click the "Upgrade available" link to see upgrade details, then click "Preview InstallPlan" button, and finally "Approve" to upgrade the Operator.
+
+=== "Operator 1.15.0 and newer"
+
+    You can actually update the Operator via the [Operator Lifecycle Manager (OLM) :octicons-link-external-16:](https://docs.redhat.com/en/documentation/openshift_container_platform/4.2/html/operators/understanding-the-operator-lifecycle-manager-olm#olm-overview_olm-understanding-olm) web interface.
+
+    Login to your OLM installation and list installed Operators for your Namespace to see if there are upgradable items:
+
+    ![image](assets/images/olm4.svg)
+
+    Click the "Upgrade available" link to see upgrade details, then click "Preview InstallPlan" button, and finally "Approve" to upgrade the Operator.
+
+## Upgrading Percona XtraDB Cluster
+
+1. Make sure that `spec.updateStrategy` option in the [Custom Resource](operator.md)
+    is set to `SmartUpdate`, `spec.upgradeOptions.apply` option is set to `Never`
+    or `Disabled` (this means that the Operator will not carry on upgrades
+    automatically).
+    
+    ```yaml
+    ...
+    spec:
+      updateStrategy: SmartUpdate
+      upgradeOptions:
+        apply: Disabled
+        ...
+    ```
+
+2. Find the **new** initial Operator installation image name with `kubectl get deploy` command (it had changed during the Operator upgrade):
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl get deploy percona-xtradb-cluster-operator -o yaml
+    ```
+
+    ??? example "Expected output"
+
+        ``` {.text .no-copy}
+        ...
+        "initContainer" : {
+          "image": "registry.connect.redhat.com/percona/percona-xtradb-cluster-operator@sha256:e8c0237ace948653d8f3e297ec67276f23f4f7fb4f8018f97f246b65604d49e6"
```

Comment:
We don't need to add extra step to update initContainer separately. We can update all params in one step.